ExampleAzureFunctionapp/ManualTriggerFunctionCS/README.md

```diff
@@ -0,0 +1,7 @@
+**Overview**
+
+This Library allows you to setup a function inside Azure Function App quickly with bare metal frame structure. 
+
+*Input Binding* - Manual Trigger using Azure Portal admin console.
+
+*Output Binding* - SendGrid, which is bound using the return type of the function in function.json
```

Comment:
Please add some instructions on how to set this up and what the expected output will be.